src/service/feedback/save.ts

```diff
@@ -0,0 +1,16 @@
+import { IFeedback } from '@/model/feedback';
+
+import { collection, addDoc } from 'firebase/firestore';
+import { firestoreDb } from '../firebaseConfig';
+
+export const saveFeedback = async (feedback: IFeedback) => {
+  try {
+    const docRef = await addDoc(collection(firestoreDb, 'feedbacks'), {
+      ...feedback
+    });
+    console.log('Document written with ID: ', docRef.id);
+  } catch (e) {
+    console.error('Error adding document: ', e);
```

Comment:
To avoid any kind of data leaking, I should avoid leaving `console.log` in production
what do you think?

Reply:
yeah, you're totally right!